odoo/tests/common.py

```diff
@@ -537,8 +537,20 @@ def executable(self):
 
         raise unittest.SkipTest("Chrome executable not found")
 
+    def _spawn_chrome(self):
+        pid = os.fork()
+        if pid != 0:
+            return pid
+        else:
+            if os.name == 'posix':
+                # prevent chrome crash because of huge memory reservation for V8
+                rlimit = resource.RLIMIT_RSS if platform.system() == 'Darwin' else resource.RLIMIT_AS
+                soft, hard = resource.getrlimit(rlimit)
+                resource.setrlimit(rlimit, (resource.RLIM_INFINITY, resource.RLIM_INFINITY))
```

Comment:
can use `os.dup2(<new>, sys.stderr.fileno())` in order to "replace" the inherited stderr by something else.